GameServerLib/Content/NavGrid.cs

```diff
@@ -156,29 +157,40 @@ public void CreateTranslation()
             }
         }
 
-        public Vector<float> TranslateToNavGrid(Vector<float> vector)
+        public Vector2 TranslateToNavGrid(Vector2 vector)
         {
+            // TODO: Remove this fast. Confusing and nonsense. Justget the grid why to create it in another function?
```

Comment:
I disagree. This function gets called in multiple translates, so having it inside the translate function would create copy and paste errors.

Reply:
could*

Reply:
All these translations give us no good. We don't need to store the map in another format.

Reply:
It's not stored in another format. This vector is the Cell X and Y. You translate the vector from -16k~+16k on the X and Y axes to 0~N, where N is table size.

Reply:
For what purpose it is needed? Just keep the data in one of the options to avoid ambiguity. That's why I think we need to remove it fast, to avoid things like `translatedCoordsDest`/`untranslatedCorrdsAfterRepos`.